bundles/fluent-en_GB/extensions/atrium-fluent-en_GB-smoke-test-kotlin_1_3/src/test/kotlin/custom/SmokeSpec.kt

```diff
@@ -8,24 +8,27 @@ import ch.tutteli.atrium.domain.builders.ExpectImpl
 import ch.tutteli.atrium.reporting.RawString
 import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
 import ch.tutteli.atrium.translations.DescriptionBasic
-import org.jetbrains.spek.api.Spek
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
 
 object SmokeSpec : Spek({
-    test("see if `toBe` can be used") {
-        expect(1).toBe(1)
-    }
-
-    test("see if `Result.isSuccess` can be used") {
-        //TODO activate after isSuccess is implemented
-        //expect(Result.success(1)).isSuccess { toBe(1) }
-    }
-
-    test("see if own assertion function without i18n can be used") {
-        expect(2).isEven()
-    }
-
-    test("see if own assertion function with i18n can be used") {
-        expect(4).isMultipleOf(2)
+    describe("Smoke tests") {
+        it("see if `toBe` can be used") {
+            expect(1).toBe(1)
+        }
+
+        it("see if `Result.isSuccess` can be used") {
+            //TODO activate after isSuccess is implemented
```

Comment:
isSucess is actually implemented by now, please activate :slightly_smiling_face:

Reply:
@assaflei would you like to do this as wel? Was not in the scope of the issue but would be nice nonetheless. Otherwise it looks good and I merge once you remove the WIP status

Reply:
I would be happy to take it, however its not a simple uncomment.
I get an unresolved reference on isSuccess,
I tried referencing ResultAssersions file containing definitions of this function but had no luck
Maybe you could provide more guidance or maybe we should take it as a different issue and let me get more experience and familiarity with the project

Reply:
btw, how do I remove the WIP status?

Reply:
No worries, I'll do it myself and post a link here so that you see what would have been necessary. 
Regarding the WIP status, I thought you had [WIP] in your PR title but you had it only in your commit message. I usually don't merge if the PR title has a [WIP] prefix as I don't know if the author intends to do more on the PR.

Reply:
Following the additional change which was necessary:
https://github.com/robstoll/atrium/commit/a46d9012b5e4a64f801d9729c11a7d8686db38d2
Actually again a wrong setup on my part. Glad you are contribution, could already fix a few internal bugs :slightly_smiling_face: